### DISCUSSION TITLE
如何制作PCL用插件

### DISCUSSION THREAD

**Discussion by AKsitaWa** [Meloong-Git/PCL#4036] [提问] [answered]
注入嘿嘿
希望PCL能开放一些api

**Comment by Chiloven945** [comment, marked as answer]
#441
已被拒

**Comment by Silverteal** [comment]
第一步，分叉存储库
第二步，自行制作插件加载器
第三步，制作插件

**Reply by MoYuan-CN** [reply]
在这种情况下，请注意遵守  [这里](https://github.com/Hex-Dragon/PCL2/discussions/3910#discussioncomment-9650420) 的协议
